.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "sparkbox"
```

Comment:
Whoops. Looks like I need to pull in the NPM dependencies for this, per [the README](https://github.com/sparkbox/eslint-config-sparkbox#eslint-config-sparkbox).

Simple enough. I'll just tack that onto a commit after merging this.